packages/features/ee/payments/api/alby-webhook.ts

```diff
@@ -0,0 +1,125 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
nit: I think we should not have app specific files in core. We should plan to move out paypal as well as alby webhook files to their respective apps only

Reply:
looks like this was done for Alby and Paypal, just stripe left to do. I will resolve this for now